### ISSUE TITLE
AFK Kick

### ISSUE THREAD

**Issue by thomasrudin** [pandorabox-io/pandorabox.io#294]
```
https://github.com/GunshipPenguin/afkkick

perhaps you might consider this mod for those who never logg out and go afk for semingly hours at a time lol ... this will help with the lag to i beleave, i have noticed several of the players seam to stay logged in and never move on the map. sitting there infront of a machine or quarrys and not playing ..

any way thought you might consider it for later on down the road or something 
```

**Comment by thomasrudin** [comment]
_Personal opinion:_
We can do that if we don't have enough slots....

**Comment by S-S-X** [comment]
Another approach that could reduce mostly-afk-players could be a little change to hunger / satiation.
Just let it go all the way down if player wont eat, this wont prevent short time afk if your satiation is full. 

It would allow pretty long afk times but results, if afk too long / not eating, would be player character killed.

**Comment by MilesBDyson** [comment]
the afkkick is configurable as well as giving a warning to the player that is about to be kicked, being semi afk and going afk to goto bed or leaving to run to the store is different .. due to the nature of the game it would not be uncommon for a player to be doing something else while running machines / quarries and such, if a player gets a warning all they have to do is move to reset the timer, setting the inactive time to say 10 - 15 minutes should be a good start.

perhaps adding an audible to the warning would be a good idea as well..

**Comment by thomasrudin** [comment]
fixed by #432 (half an hour afk time for now)
